packages/babel-traverse/test/fixtures/conversion/implicit-var-arguments/output.js

```diff
@@ -0,0 +1,20 @@
+var _arguments = typeof arguments === "undefined" ? void 0 : arguments;
+
+var arguments = [1, 2, 3];
+
+var arr = function arr() {
+  return _arguments[0];
```

Comment:
This looks wrong, it should still be `arguments[0]`.

Reply:
Yeah, It's I will fix it.